.github/workflows/deploy.yml

```diff
@@ -27,9 +27,16 @@ jobs:
       - name: Git clone the repository
         uses: actions/checkout@v3
 
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+          cache: pip   # caching pip dependencies
+
       - name: Build static assets
         run: |
-           ./build.sh
+           pip install -r requirements_dev.txt
+           python3 build.py
```

Comment:
do we want to install dev requirements?

Reply:
@cyrusdobbs Yes, the build for the flask assets depends on the requirements.

Reply:
Should it be in dev or just the standard requirements then?

Reply:
I think requirements_dev as it's only used in the CI context